migrate_nidirect_node/migrate_nidirect_node_nidirect_contact/src/Plugin/migrate/source/NIDirectContactNodeRevisionSource.php

```diff
@@ -18,6 +18,7 @@
  *
  * @MigrateSource(
  *   id = "nidirect_contact_node_revision_source",
+ *   source_module = "node"
```

Comment:
I remember migrate was sometimes grumpy about this being missing in the past. I guess it's now settled on needing to be there

Reply:
yeah, it only really started complaining about it once I'd reinstalled the modules and ran migrate status